features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/services/login/basic.jag

```diff
@@ -0,0 +1,57 @@
+<%
+    var log = new Log("Services login app");
+    var url = 'https://localhost:9443/client-registration/v0.14/register';
+    var data = { "callbackUrl": "localhost", "clientName": request.getParameter("application", "UTF-8") , "owner": "admin", "grantType": "password refresh_token", "saasApp": true };
+    var result = post(url, JSON.stringify(data) , {
+        "Authorization": "Basic YWRtaW46YWRtaW4=",
+        "Content-Type": "application/json"
+    }, "json");
+
+    //curl -k -d "grant_type=password&username=admin&password=admin&scope=apim:subscribe" -H "Authorization: Basic SGZFbDFqSlBkZzV0YnRyeGhBd3liTjA1UUdvYTpsNmMwYW9MY1dSM2Z3ZXpIaGM3WG9HT2h0NUFh" https://localhost:8243/token
```

Comment:
Shall we remove this comment as well ? I think we don't need a sample curl command here

Reply:
resolved